data/DataHandler.php

```diff
@@ -0,0 +1,152 @@
+<?php
+//==================================
+// Class Definition of DataHandler
+//==================================
+
+require __DIR__.'/../utility/Equipment.php';
+require __DIR__.'/../utility/Room.php';
+
+define ('WAREHOUSE', 'warehouse');
+define("MAX_SPACE", 1000);
```

Comment:
If these two constants belong to DataHandler?
If yes, define them inside the class (as a class constants)
If not, think about a separate place to have those defined and organized in